scripts/install.ps1

```
@@ -1,29 +1,315 @@
+.EXAMPLE
+  pwsh -File scripts\install.ps1 -Browser both -DryRun
+#>
+
+[CmdletBinding()]
+param(
+  [ValidateSet('chrome', 'brave', 'edge', 'both')]
+  [string]$Browser,
+
+  # NOT named $Profile — that is a PowerShell automatic variable.
+  [Alias('Profile')]
+  [string]$ProfileName = 'Default',
```

PSScriptAnalyzer (1.25.0)

[warning] The parameter 'ProfileName' has been declared but not used.

(PSReviewUnusedParameter)